playwright.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig({
   snapshotPathTemplate: './tests/snapshots/{arg}{ext}',
   timeout: 30 * 1000,
   expect: {
-    timeout: 5000,
+    timeout: 8000,
```

Comment:
This seems long. Are we sure we need to do this?

Reply:
Reverted this, had changed it to test an intermittent issue with the visual tests.